examples/src/main/scala/org/apache/spark/examples/HBaseTest.scala

```diff
@@ -36,7 +36,7 @@ object HBaseTest {
     // Initialize hBase table if necessary
     val admin = new HBaseAdmin(conf)
     if (!admin.isTableAvailable(args(0))) {
-      val tableDesc = new HTableDescriptor(args(0))
+      val tableDesc = new HTableDescriptor(TableName.valueOf(args(0)))
```

Comment:
Do you happen to know how long ago this constructor was added? I want to figure out of this makes it incompatible with any HBase >= 0.98.7, which is presumably the earliest version kind of 'supported' by the examples.

Are there other deprecations in the HBase examples that can be improved? I suspect the examples were written for HBase ~0.94.x